loop/routes/rooms.js

```diff
@@ -481,4 +481,50 @@ module.exports = function (apiRouter, conf, logError, storage, auth,
       );
     });
   });
+
+  /**
+   * Remove given rooms
+   **/
+  apiRouter.patch('/rooms', auth.requireHawkSession,
+    validators.requireParams('deleteRoomTokens'), function(req, res) {
+      var status = 200;
+      var roomTokens = req.body.deleteRoomTokens;
+      storage.getUserRooms(req.user, function(err, userRooms) {
+        if (res.serverError(err)) return;
+
+        var roomsToDelete = userRooms.filter(function(room) {
+          return roomTokens.indexOf(room.roomToken) !== -1;
+        }).map(function(room) {
+          return room.roomToken;
+        });
+
+        if (roomsToDelete.length === 0) {
+          // No rooms founds
+          status = 404;
```

Comment:
This also means a PATCH on `/rooms` with an empty list will return a 404. I'm not sure we want that.